guides/common/modules/ref_managing-packages-on-the-base-operating-system.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Managing_Packages_on_the_Base_Operating_System_{context}"]
+= Managing packages on the base operating system of {ProjectServer} or {SmartProxyServer}
+
+{Project} provides the `{foreman-maintain} packages` command to install and update packages on the base operating system of your {ProjectServer} or {SmartProxyServers}.
+
+[WARNING]
+====
+Installing and updating individual packages with `yum` is not supported.
+While {Project} provides the `{foreman-maintain} packages unlock` command to bypass `{foreman-maintain} packages` and enable package management with `yum`, this can lead to package inconsistencies and compromise the stability of the system.
+Always use `{foreman-maintain} packages` to manage packages.
+====
+
+After you install or update a package on the base operating system, running `{foreman-installer}` afterwards is required.
```

Comment:
```suggestion
You must run `{foreman-installer}` after you install or update a package on the base operating system.
```

alternative:

```suggestion
You must run `{foreman-installer}` after installing or updating a package on the base operating system.
```

Reply:
I actually agonized over this for quite some time. Writing "you must run" is not entirely true because if you manage the packages in the supported way (`foreman-maintain packages`), `foreman-installer` is run automatically as part of `foreman-maintain packages`.

But if you manage packages in the non-supported way (unlock + dnf/yum), you might not know that `foreman-installer` is required too.

While we don't support the other way but in reality, some users do it anyway. In which case we don't want them to break their Foreman _too much_. So I was looking for a wording that would support that scenario -- while being careful not to encourage it.

While I know that the wording I used is not great, I also feel that I can't accept your suggestions.

Reply:
I look at the whole paragraph again and reworded it slightly to make it sound more natural. I hope it helps at least a little bit.